assets/src/application/ui/forms/ee-form/fields/Group.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { FieldProps } from '../types';
+import Field from './Field';
+import FieldRenderer from '../renderers/FieldRenderer';
+import useShouldBeVisible from '../hooks/useShouldBeVisible';
+
+const Group: React.FC<FieldProps> = ({ subFields, label, name: groupName, before, after, conditions }) => {
+	const visible = useShouldBeVisible(conditions, groupName);
+	if (subFields.length && visible) {
+		return (
+			<div className='field-group-wrapper'>
+				<h5>{label}</h5>
+				{before}
+				<div className='field-group-items'>
+					{subFields.map(({ name: fieldname, fieldType, ...props }, i) => {
+						const name = `${groupName}.${fieldname}`;
+						const position = i % 2 === 0 ? 'left' : 'right';
```

Comment:
??? 

This appears to be an assumption about layout, with the position baked into the logic (ie: all groups will have two columns).

Why are even number items always on the right? What if a form design doesn't want two columns?
As well, what if a language is `rtl` ? (that can likely be handled via css, specifically flexbox)

Can layout decisions like this be abstracted away from the base form components? For example, in the EDTR v1, the `assets\ZZZ\components\form\layouts\two-column-admin` folder contains components that can be used to apply a two column design layout to your form, and although no other layout strategies exist, the idea was to have a similar set of components that would apply different layouts to a form. So the idea was that you could completely change the layout for a form simply by importing a different layout strategy, either manually:

```js
import { twoColumnAdminFormLayout } from '@eventespresso/components';
const { FormRow, FormSection, FormWrapper } = twoColumnAdminFormLayout;
```

or via props

```js
const { FormRow, FormSection, FormWrapper } = props.layoutStrategy;
```

with all of the other code in the component exactly the same.

This doesn't have to work exactly the same way that I did things in EDTR v1, as I will leave that up to your expertise to solve, but I can guarantee that there will be different requirements for form layouts in different areas of the plugin, so anything we build should have some ability to be versatile.

Reply:
Yeah, I wasn't sure about how to handle fields inside a group, so I thought to put 2 fields in a row. The reason for making it that way is again that JSON config oriented approach where you cannot import those layout components and can be a problem with those D&D form builders.

Reply:
I have removed that `left` and `right` class but would like to discuss with you about how we want to support the layout, especially for the form config based approach.

Reply:
maybe the JSON config can contain a field for setting the layout?
and maybe allow each section to set a different layout (but default to the previously set layout),
that's how our PHP form system work actually.